app/state/reducers/recurringReservationsReducer.spec.js

```diff
@@ -0,0 +1,420 @@
+import { expect } from 'chai';
+import { actionTypes as formActions } from 'redux-form';
+
+import actions from 'actions/recurringReservations';
+import recurringReservationsReducer, { populateReservations } from './recurringReservationsReducer';
+
+describe('state/recurringReservations', () => {
+  describe('reducer', () => {
+    const reducer = recurringReservationsReducer;
```

Comment:
- [ ] Instead of doing this you could just import it as `reducer`: `import reducer, { populateReservations } from './recurringReservationsReducer';`.